service/src/main/java/bio/terra/cbas/controllers/RunsApiController.java

```diff
@@ -85,6 +90,13 @@ public ResponseEntity<Void> postRunResults(RunResultsRequest body) {
     CbasRunStatus resultsStatus = CbasRunStatus.fromCromwellStatus(body.getState().toString());
     List<String> failures = body.getFailures();
 
+    Counter counter =
```

Comment:
This looks similar to how we did a custom metric, but I want to point out that Spring Boot 3 brings a much more interesting and expressive `Observation` interface, which I think is think will be valuable for getting tracing as well as metrics gathering.  We haven't used it much yet but there's a near-landing PR where we start trying it out under way here: https://github.com/DataBiosphere/terra-workspace-data-service/pull/461/files#r1460108734